english/snuffbox/explore/explore.md

```diff
@@ -0,0 +1 @@
+After the titanic sneaze rocked baseball sized holes in the marshmellow walls surrounding you. A curious thought enters your head, 'I wonder what is in them?' you ask yourself. Cautiously you aproche one of them and slowly reach out towards the nearest hole. The sweat on your brow is telling you that this is how people lose arms in horror stories, but you press on. As you hand slids into the snot created hole you feel the slime on your arm and suddenly you AWAKE! Your alarm is blaring and you realize you have drooled all over your bed again! 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.